packages/control-property-editor-common/tsconfig-base.json

```diff
@@ -0,0 +1,28 @@
+{
+  "compilerOptions": {
+    "rootDir": ".",
+    "composite": true,
+    "declaration": true,
+    "declarationMap": true,
+    "incremental": true,
+    "outDir": "dist",
+    "target": "ES2021",
+    "module": "commonjs",
+    "moduleResolution": "node",
+    "esModuleInterop": true,
+    "resolveJsonModule": true,
+    "experimentalDecorators": true,
+    "sourceMap": true,
+    "preserveSymlinks": true,
+    "types": ["node", "jest"],
+    "skipLibCheck": true,
+    "preserveConstEnums": true,
+    "importsNotUsedAsValues": "remove",
+    "baseUrl": ".", // This must be specified if "paths" is.
```

Comment:
Comments are not allowed in JSON. Please remove

Reply:
removed